### DISCUSSION TITLE
Windows 7 binary

### DISCUSSION THREAD

**Discussion by Vort** [yggdrasil-network/yggdrasil-go#1162] [Show and tell]
Hello.

I made Yggdrasil 0.5.8 build, compatible with Windows 7 in case someone needs it:
[yggdrasil-develop-0.5.8-x64_win7.zip](https://github.com/user-attachments/files/16611064/yggdrasil-develop-0.5.8-x64_win7.zip)

Nothing was cut or downgraded, I just used Go 1.21.13 compiler with 9e43850a3298a9b8b1162ba0033d4c53f8637571 commit reverted.

**Comment by ghost** [comment]
And x86, please.

**Reply by Vort** [reply]
Can you check if I made it correctly?
[yggdrasil-develop-0.5.8-x86_win7.zip](https://github.com/user-attachments/files/16612801/yggdrasil-develop-0.5.8-x86_win7.zip)
